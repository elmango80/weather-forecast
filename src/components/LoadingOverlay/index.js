import React from "react";
import ReactLoading from "react-loading";

import { EuiOverlayMask } from "@elastic/eui";

export default function LoadingOverlay() {
  return (
    <>
      <EuiOverlayMask headerZindexLocation="below">
        <ReactLoading className="m-auto" type="bars" color="#f5df4d" />
      </EuiOverlayMask>
    </>
  );
}
