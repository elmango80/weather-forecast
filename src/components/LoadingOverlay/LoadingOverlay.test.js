import React from "react";
import "@testing-library/jest-dom";
import { shallow } from "enzyme";

import LoadingOverlay from "./index";

describe("Test <LoadingOverlay />", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<LoadingOverlay />);

    expect(wrapper).toMatchSnapshot();
  });
});
