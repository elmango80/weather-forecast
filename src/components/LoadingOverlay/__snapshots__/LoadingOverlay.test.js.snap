// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <LoadingOverlay /> renders correctly 1`] = `
<Fragment>
  <EuiOverlayMask
    headerZindexLocation="below"
  >
    <Loading
      className="m-auto"
      color="#f5df4d"
      delay={0}
      height={64}
      type="bars"
      width={64}
    />
  </EuiOverlayMask>
</Fragment>
`;
